src/po/sr.po

```diff
@@ -4658,6 +4508,26 @@ msgstr "Vim Упозорење"
 msgid "shell returned %d"
 msgstr "командно окружење је вратило %d"
 
+msgid "E278: Cannot put a terminal buffer in a popup window"
+msgstr "E278: Терминалски бафер не може да се стави у искачући прозор"
+
+#, c-format
+msgid "E997: Tabpage not found: %d"
+msgstr "E997: Картица није пронађена: %d"
```

Comment:
I have also seen "jezicak" for tab, though that's a horrendous translation and there definitely are no tongues involved.